<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>493</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>744</width>
    <height>493</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>- ALTA SOCIO -</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelNombre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NOMBRE</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelApellido">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>APELLIDO</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDNI">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DNI</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDomicilio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DOMICILIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTelLinea">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TEL.LINEA</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCelular">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CELULAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEmail">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>E-MAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRedSocial">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RED SOCIAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelObservacion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OBSERVACIÓN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSexo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNombre">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditApellido">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDNI">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDomicilio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLocalidad">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LOCALIDAD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLocalidad">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCelular">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTelLinea">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhEmailCharactersOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRedSocial">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAltaSalir">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAltaConfirmar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIRMA</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMostrarPrecio">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditObservacion">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxAtividades">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>131</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Actividades</string>
    </property>
    <widget class="QRadioButton" name="radioButtonMusculacion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MUSCULACIÓN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFitness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FITNESS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonPaseLibre">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PASE LIBRE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonBoxeo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>BOXEO</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonPilates">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PILATES</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSexo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButtonMasculino">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MASCULINO</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFemenino">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FEMENINO</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButtonPromo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PROMO</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAltaImagenLogo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo_orbis.jpg</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
